docs/reference/commandline/daemon.md

```diff
@@ -50,6 +50,7 @@ weight = -1
       --log-driver="json-file"               Default driver for container logs
       --log-opt=[]                           Log driver specific options
       --mtu=0                                Set the containers network MTU
+      --no-color=false                       Prevents the daemon from logging in color
```

Comment:
Ignore the other ones, I'll deal with that (and the other doc pages) in a new PR - just fix-up the new option you're adding.